### PR TITLE
Show ID everywhere

### PR DESCRIPTION
**Pull request by untitaker**
Given that the ID is now persistent, it makes sense to always print it.

If I create a task and want to edit it immediately, I first have to list
all tasks to get the ID.

### CODE REVIEW

**Comment by hobarrera** [comment]
Thanks, I was thinking the same thing today.

**Comment by untitaker** [comment]
I'll fix tests

**Comment by hobarrera** [comment]
Oh, yeah, now I remember why I didn't implement it; when reading from a file, the `id` is not set. We'd need to implement this somewhere (IIRC, `new` is especially painful).

File: todoman/model.py [hunk 1]
```diff
@@ -781,6 +787,16 @@ def flush(self):
...
 
+    def save(self, todo, list_):
+        todo.save(list_)
+        path = os.path.join(list_.path, todo.filename)
+        self.cache.expire_file(path)
+        mtime = _getmtime(path)
+        self.cache.add_file(list_.name, path, mtime)
+        id = self.cache.add_todo(todo, path)
+        self.cache.save_to_disk()
+        todo.id = id
```
Comment (untitaker) [inline todoman/model.py]: This just patches the `id` onto `FileTodo`. Is this OK?

Comment (hobarrera) [inline todoman/model.py]: Yeah, I can't really think of any other way to do this.

**Comment by untitaker** [comment]
`lastrowid` (which we return in `add_todo`) is only available with a cursor

**Comment by hobarrera** [comment]
Oh, okay. LGTM, feel free to merge this.

**Pull request merged**